Opening DXF file results in 500 error

I tried opening two dxf files, a 3D dxf and a 2D dxf file.
Both result in a 500 error.
Opening both files in the F3D desktop application works fine.

The sample files I used:
[samples#1146.zip](https://github.com/user-attachments/files/28324646/samples.1146.zip)

The Log:
```
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d]   
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d] ActionView::Template::Error (uninitialized constant ActionView::Renderer::ImageTag)
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d] Caused by: NameError (uninitialized constant ActionView::Renderer::ImageTag)
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d] 
23:53:39 rails.1              | Information for: ActionView::Template::Error (uninitialized constant ActionView::Renderer::ImageTag):
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d]     34:     <% if (handler = FileHandlers::Base.handlers_for(environment: :browser, load_file: @file)&.first) %>
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d]     35:       <%= render handler.component.new(file: @file) %>
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d]     36:     <% elsif @file.has_render? %>
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d]     37:       <%= render Renderer::ImageTag.new(file: @file) %>
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d]     38:     <% end %>
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d]     39:     <% if @file.notes %>
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d]     40:       <%= card(:secondary, t(".notes_heading")) do %>
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d]   
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d] app/views/model_files/show.html.erb:37
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d] app/controllers/application_controller.rb:134:in 'ApplicationController#switch_locale'
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d] 
23:53:39 rails.1              | Information for cause: NameError (uninitialized constant ActionView::Renderer::ImageTag):
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d]   
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d] app/views/model_files/show.html.erb:37
23:53:39 rails.1              | [1f8e595d-09b0-4cb0-b3dd-5876db4cd03d] app/controllers/application_controller.rb:134:in 'ApplicationController#switch_locale'
```

--- a/app/views/model_files/show.html.erb
+++ b/app/views/model_files/show.html.erb
@@ -34,7 +34,7 @@
     <% if (handler = FileHandlers::Base.handlers_for(environment: :browser, load_file: @file)&.first) %>
       <%= render handler.component.new(file: @file) %>
     <% elsif @file.has_render? %>
-      <%= render Renderer::ImageTag.new(file: @file) %>
+      <%= image_tag model_model_file_path(@file.model, @file, format: @file.extension, derivative: "render"), style: "width: 100%", alt: @file.name %>
     <% end %>
     <% if @file.notes %>
       <%= card(:secondary, t(".notes_heading")) do %>

